README.md

```diff
@@ -22,7 +28,120 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+### `FastStats::Means`
+
+Collect means for multiple metrics:
+
+### Initialize
```

Comment:
What about organizing the section headers like this:
```
## Usage
   
### `FastStats::Means`

Collect means for multiple metrics.

#### Initialize
#### Instance Methods
#### Example


### `FastStats::Mean`

Build up the arithmetic and geometric means for a single metric.

#### Initialize
#### Instance Methods
#### Example
```

And maybe swap the order and do single first, then multiple?  To go in order of increasing complexity?

Reply:
@andreafrost That is the current order of subsection headers??

Reply:
I was referring to using `####` instead of `###` for all 3 subsections (`Initialize, Instance Methods, Example`), which I see you have now updated :)